tests/system/Config/fixtures/.env

```diff
@@ -6,4 +6,5 @@ NULL=
 
 SimpleConfig.onedeep=baz
 SimpleConfig.default.name=ci4
-simple.name=simpleton
\ No newline at end of file
+simple.name=simpleton
+impleconfig.shortprefix=wht
```

Comment:
Not sure how your test is passing when you're missing the `s` from SimpleConfig...